unmarshaller_test.go

```diff
@@ -34,102 +39,138 @@ c,d,e
 	reader.FieldsPerRecord = -1
 	c := &Config{Holder: sample{}}
 	um, err := c.NewUnmarshaller(reader)
-	if err != nil {
-		t.Fatalf("Error calling NewUnmarshaller: %#v", err)
-	}
+	require.NoError(t, err)
 
 	obj, err := um.Read()
-	if err != nil {
-		t.Fatalf("Error calling Read(): %#v", err)
-	}
-	if obj.(sample).FieldA != "a" || obj.(sample).FieldB != "b" {
-		t.Fatalf("Unepxected result from Read(): %#v", obj)
-	}
+	require.NoError(t, err)
+	assert.Equal(t, sample{"a", "b"}, obj, "Unexpected result from Read()")
 
 	obj, err = um.Read()
-	if err != nil {
-		t.Fatalf("Error calling Read(): %#v", err)
-	}
-	if obj.(sample).FieldA != "c" || obj.(sample).FieldB != "d" {
-		t.Fatalf("Unepxected result from Read(): %#v", obj)
-	}
+	require.NoError(t, err)
+	assert.Equal(t, sample{"c", "d"}, obj, "Unexpected result from Read()")
 
 	obj, err = um.Read()
-	if err != io.EOF {
-		t.Fatalf("Unepxected result from Read(): (%#v, %#v)", obj, err)
-	}
+	require.Equal(t, io.EOF, err, "Expected io.EOF")
 }
 
 func Test_Read_ErrorLineNumbers(t *testing.T) {
-    t.Parallel()
+	t.Parallel()
 
 	um, err := NewUnmarshaller(sample{}, csv.NewReader(strings.NewReader(brokenCSV)))
-	if err != nil {
-		t.Fatalf("Failed to allocate Unmarshaller: %s", err.Error())
-	}
-
+	require.NoError(t, err)
 
 	// Lines 4 & 5 have errors.
 
 	var rec interface{}
 	// Header is line 1
-	_, err = um.Read()			// line 2
-	_, err = um.Read()			// line 3
-	_, err = um.Read()			// line 4
+	_, err = um.Read() // line 2
+	_, err = um.Read() // line 3
```

Comment:
Minor: we're never using this error value.
```suggestion
	um.Read() // line 3
```

Reply:
Alternatively, we could `assert.NoError` on lines 2 and 3.